src/themes/ionic.mixins.scss

```diff
@@ -157,16 +153,44 @@
   }
 }
 
-$include-rtl: true !default;
+$app-direction: null !default;
+
+@mixin multi-dir() {
+  @if $app-direction == null {
+    $root: #{&};
+    @at-root [dir="ltr"], [dir="rtl"] {
```

Comment:
Why do we need to specify `dir` if the rule applies to both? Also will this work without `@at-root` being in front of `[dir="rtl"]`?